editor/blocks/quote/index.js

```diff
@@ -0,0 +1,42 @@
+const Editable = wp.blocks.Editable;
+const { html } = wp.blocks.query;
+
+wp.blocks.registerBlock( 'core/quote', {
+	title: 'Quote',
+	icon: 'format-quote',
+	category: 'common',
+
+	attributes: {
+		value: html( 'blockquote > p' ),
```

Comment:
We will probably want to map the values in `value` to its of `<p>` component as well right?